packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1257,6 +1257,14 @@ export default class Meeting extends StatelessWebexPlugin {
     return this.joinedWith?.state === 'JOINED';
   }
 
+  /**
+   * Returns whether this meeting is a Locus CALL
+   * @returns {Boolean}
+   */
+  isCall() {
```

Comment:
For clarity can we name it as `isLocalCall` ?

Reply:
done